src/orion/client/__init__.py

```diff
@@ -331,14 +342,15 @@ def workon(
 
     try:
         setup_storage(storage={"type": "legacy", "database": {"type": "EphemeralDB"}})
-
+        # TODO: A bit weird that we pass the kb to both the experiment and exp client...
```

Comment:
It's only passed to the experiment here, no?

Reply:
Yeah outdated todo. Removed in b0f9de3a0605afaf21973f510ba160da8146cd00